src/com/bosiy/Triangle.java

```diff
@@ -0,0 +1,31 @@
+package com.bosiy;
+
+public class Triangle {
+    private final int sideA;
+    private final int sideB;
+    private final int sideC;
+
+    public Triangle(int sideA, int sideB, int sideC) {
```

Comment:
or what will be if I'll create unrealistic triangle?

for ex. first side = 1, second side = 1, and third side = 328

Reply:
fixed